### PR TITLE
Update Virtualisation.md

### PR DESCRIPTION
**Pull request by einsteinagogo**
Typo in Link

### CODE REVIEW

**Comment by closebracket** [comment]
Did you want this reviewed/approved or is there more to come?

**Comment by einsteinagogo** [comment]
Okay, can you review, now I've fixed all the links.

File: Ethics_Law_Professionalism.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 9. [Linux Distributions](README.md#linux)
 10. [Electronics](Electronics.md)
 11. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
-    - [Andrew Hancock, VMware vExpert PR](Virtualisation.md#andrewhancock)
+    - [Andrew Hancock, VMware vExpert PRO](Virtualisation.md#andrewhancock)
```
Comment (closebracket) [inline Ethics_Law_Professionalism.md:18]: Cool

File: Programming.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
 9. [Linux Distributions](README.md#linux)
 10. [Electronics](Electronics.md)
 11. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
-    - [Andrew Hancock, VMware vExpert PR](Virtualisation.md#andrewhancock)
+    - [Andrew Hancock, VMware vExpert PRO](Virtualisation.md#andrewhancock)
```
Comment (closebracket) [inline Programming.md:18]: Cool

File: README.md [hunk 3]
```diff
@@ -15,7 +15,7 @@
 9. [Linux Distributions](#linux)
 10. [Electronics](Electronics.md)
 11. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
-    - [Andrew Hancock, VMware vExpert PO](Virtualisation.md#andrewhancock)
+    - [Andrew Hancock, VMware vExpert PRO](Virtualisation.md#andrewhancock)
```
Comment (closebracket) [inline README.md:18]: Cool

File: Virtualisation.md [hunk 4]
```diff
@@ -15,7 +15,7 @@
 9. [Linux Distributions](README.md#linux)
 10. [Electronics](Electronics.md)
 11. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
-    - [Andrew Hancock, VMware vExpert PR](Virtualisation.md#andrewhancock)
+    - [Andrew Hancock, VMware vExpert PRO](Virtualisation.md#andrewhancock)
```
Comment (closebracket) [inline Virtualisation.md:18]: Cool

File: Virtualisation.md [hunk 5]
```diff
@@ -44,7 +44,7 @@ Type 1 (Bare Metal Hypervisors)
 4. [XCP-ng](https://xcp-ng.org/) - Based on Xen Server - Available on GitHub.
 5. [Citrix Hypervisor](https://www.citrix.com/en-gb/products/citrix-hypervisor/) - Originally based on Xen Server. Commercial version by Citrix.
 6. [Microsoft Windows Server 2019](https://www.microsoft.com/en-gb/windows-server) - Hyper-V can be enabled as a role in Windows Server. This is a Type 1 hypervisor. - Available for free if a Student of The Department of Computer Science and Technology.
-7. [Microsoft Hyper-V Server](https://www.microsoft.com/en-us/evalcenter/evaluate-hyper-v-server-2019_) - Not to be confused with Windows Server. This is a Type 1 Hypervisor, and is just server core Hyper-V with no GUI. - Available for free if a Student of The Department of Computer Science and Technology.
+7. [Microsoft Hyper-V Server](https://www.microsoft.com/en-us/evalcenter/evaluate-hyper-v-server-2019) - Not to be confused with Windows Server. This is a Type 1 Hypervisor, and is just server core Hyper-V with no GUI. - Available for free if a Student of The Department of Computer Science and Technology.
```
Comment (closebracket) [inline Virtualisation.md:47]: Link is working. Thanks for correcting.

File: Virtualisation.md [hunk 6]
```diff
@@ -70,7 +70,9 @@ Type 2
...
+Andrew is the VMware Specialist in Computer Science & Technology, responsible for the VMware architecture, "watering & feeding" The Computer Science and Technology VMware vSphere infrastructure. (based on VMware vSphere). He also has a long list of VMware and Virtualisations articles, a passionate VMware Technical Architect, and VMware vExpert PRO. He has won over 40 [Experts Exchange Awards](https://www.experts-exchange.com/members/hanccocka.html) and is Overall Leader at [Experts Exchange.](https://www.experts-exchange.com/). Experts Exchange - The Original Tech Community™ - Formed in 1996, Experts Exchange (EE) is one of the oldest online communities in the world. For 25 years, tech problem-solvers worldwide have gathered on EE to share knowledge and help each other succeed.
```
Comment (closebracket) [inline Virtualisation.md]: I'm not sure many people will understand the CS&T references given the lack of association with UoH. Could you please rephrase the paragraphs?

File: CyberSecurity.md [hunk 7]
```diff
@@ -15,7 +15,7 @@
 9. [Linux Distributions](README.md#linux)
 10. [Electronics](Electronics.md)
 11. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
-    - [Andrew Hancock, VMware vExpert](Virtualisation.md#andrewhancock)
+    - [Andrew Hancock, VMware vExpert PRO](Virtualisation.md#andrewhancock)
```
Comment (closebracket) [inline CyberSecurity.md:18]: Cool

File: CyberSecurity.md [hunk 8]
```diff
@@ -22,7 +22,7 @@
 
 # Cyber Security List <a name="cyber"></a>
 
-> ## Disclaimer: 
+> ### Disclaimer:
```
Comment (closebracket) [inline CyberSecurity.md:25]: Good shout. Probably worth updating the disclaimer as well. I'll get onto that.

File: Electronics.md [hunk 9]
```diff
@@ -15,7 +15,7 @@
 9. [Linux Distributions](README.md#linux)
 10. [Electronics](Electronics.md)
 11. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
-    - [Andrew Hancock, VMware vExpert PRO](Virtualisation.md#andrewhancock)
+    - [Andrew Hancock, VMware vExpert PR](Virtualisation.md#andrewhancock)
```
Comment (closebracket) [inline Electronics.md]: Seems like it went form PRO to PR?

File: Electronics.md [hunk 10]
```diff
@@ -15,7 +15,7 @@
 9. [Linux Distributions](README.md#linux)
 10. [Electronics](Electronics.md)
 11. [Hypervisors and virtual machines](Virtualisation.md#virtualisation)
-    - [Andrew Hancock, VMware vExpert PR](Virtualisation.md#andrewhancock)
+    - [Andrew Hancock, VMware vExpert PRO](Virtualisation.md#andrewhancock)
```
Comment (closebracket) [inline Electronics.md:18]: And it was fixed. No problem.

File: Virtualisation.md [hunk 11]
```diff
@@ -68,13 +68,12 @@ Type 2
...
-1. [How to Backup an ESXi installation on an USB Flash Drive or SD card for security or redundancy.](http://vexpert.me/BackupESXi)
-2. [VMware ESX/ESXi Backup Guide](http://vexpert.me/BackupGuide)
+His articles are reproduced here for your reading pleasure. If you want to discuss anything VMware, then ask him! He will only be too happy to oblige. All this software is FREE here - [VMware IT Academy Program](https://www.vmware.com/uk/company/it-academy.html) 
+1. [How to Backup an ESXi installation on an USB Flash Drive or SD card for security or redundancy.](https://www.experts-exchange.com/A_5409.html)
```
Comment (closebracket) [inline Virtualisation.md:75]: Groovy! Thanks for fixing!

File: Virtualisation.md [hunk 12]
```diff
@@ -68,13 +68,12 @@ Type 2
...
+2. [VMware ESX/ESXi Backup Guide](https://www.experts-exchange.com/A_5414.html)
 3. [HOW TO: Improve the transfer rate of a Physical to Virtual (P2V) Virtual to Virtual Conversion (V2V) using VMware vCenter Converter Standalone 5.0](https://www.experts-exchange.com/articles/9131/HOW-TO-Improve-the-transfer-rate-of-a-Physical-to-Virtual-P2V-Virtual-to-Virtual-Conversion-V2V-using-VMware-vCenter-Converter-Standalone-5-0.html)
```
Comment (closebracket) [inline Virtualisation.md:77]: Groovy! Thanks for fixing!

File: Virtualisation.md [hunk 13]
```diff
@@ -200,12 +199,12 @@ His articles are reproduced here for your reading pleasure. If you want to discu
...
 129. [Part 21: HOW TO: Deploy and Install VMware vSphere vCenter Server 7.0 (VCSA 7.0)](https://www.experts-exchange.com/articles/35011/HOW-TO-Deploy-and-Install-VMware-vSphere-vCenter-Server-7-0-VCSA-7-0.html)
 130. [Part 22: HOW TO: Install and Connect a VMware vCenter Server 7.0 in linked mode.](https://www.experts-exchange.com/articles/35052/HOW-TO-Install-and-Connect-a-VMware-vCenter-Server-7-0-in-linked-mode.html)
-131. [Part 23: HOW TO: BOOT VMware vSphere Hypervisor 7.0 (ESXi 7.0 ARM) from an iSCSI LUN for the Raspberry Pi 4](http://vexpert.me/iSCSI-BOOT-from-ESXi-ARM)
-132. [Part 24: HOW TO: Update VMware ESXi 7.0U2 to ESXi 7.0U2a direct from VMware.](http://vexpert.me/1dW)
-133. [Part 25: HOW TO: Update VMware ESXi 7.0U2 to ESXi 7.0U2a in 5 easy steps](http://vexpert.me/1dX)
+131. [Part 23: HOW TO: BOOT VMware vSphere Hypervisor 7.0 (ESXi 7.0 ARM) from an iSCSI LUN for the Raspberry Pi 4](https://www.experts-exchange.com/articles/35132/HOW-TO-BOOT-VMware-vSphere-Hypervisor-7-0-ESXi-7-0-ARM-from-an-iSCSI-LUN-for-the-Raspberry-Pi-4.html)
```
Comment (closebracket) [inline Virtualisation.md:205]: Groovy! Thanks for fixing!

File: Virtualisation.md [hunk 14]
```diff
@@ -215,25 +214,25 @@ In this series, which compliments the written articles above, Andy Hancock demon
...
-16. [Part 16: HOW TO: Update VMware ESXi 7.0U2 to ESXi 7.0U2a direct from VMware.](http://vexpert.me/1eb)
-17. [Part 17: HOW TO: Update VMware ESXi 7.0U2 to ESXi 7.0U2a in 5 easy steps.](http://vexpert.me/1e8)
-18. [Part 18: HOW TO: Update VMware ESXi 7U1 (7.0.1) to VMware ESXi 7U2a (7.0.2) using an ISO image.](http://vexpert.me/1e9)
-19. [Part 19: HOW TO: Update VMware ESXi 7.0U1 to ESXi 7.0U2a using VMware vSphere Lifecycle Manager.(vLCM)](http://vexpert.me/1ea)
-20. [Part 20: HOW TO: Rollback VMware vSphere Hypervisor 7.0 U2a using VMware Hypervisor Recovery Mode.](http://vexpert.me/1e5)
-21. [Part 21: HOW TO: Deploy and Install VMware vSphere vCenter Server 7.0 (VCSA 7.0)](http://vexpert.me/1er)
-22. [Part 22: HOW TO: Add VMware vSphere Hypervisor ESXi 7.0 Hosts to VMware vSphere vCenter Server 7.0](http://vexpert.me/1es)
+1. [Part 1: HOW TO: Install and Configure VMware vSphere Hypervisor 7.0 U2 (ESXi 7.0.2).](https://www.experts-exchange.com/videos/78972/HOW-TO-Install-and-Configure-VMware-vSphere-Hypervisor-7-0-U2-ESXi-7-0-2.html)
```
Comment (closebracket) [inline Virtualisation.md:217]: Groovy! Thanks for fixing!

**Review by closebracket — CHANGES_REQUESTED** [review]
Hi @einsteinagogo, this is a huge update which is massively appreciated. Just one change that I can see to the paragraphs in Virtualisation.md. Once that's done, you're ready to be approved.

**Comment by einsteinagogo** [comment]
> Hi @einsteinagogo, this is a huge update which is massively appreciated. Just one change that I can see to the paragraphs in Virtualisation.md. Once that's done, you're ready to be approved.

whats the issue with the Paragraphs, I'm not seeing an issue?

**Comment by einsteinagogo** [comment]
Okay, I had to expand all the comments, to check what you were referring to!

File: Virtualisation.md (hunk 6 above)
Comment (einsteinagogo) [reply Virtualisation.md]: ah! I can see this now, had to expand all comment fields! standby!

**Review by einsteinagogo — COMMENTED** [review]
okay, I think we are now done.

**Comment by closebracket** [comment]
> okay, I think we are now done.

Agreed. Good inclusion of EE and fair link to your Ltd given your contributions. Thanks for updating. Will merge.

**Review by closebracket — APPROVED** [review]
Changes made as requested. Approving for merge.

**Pull request merged**